### PR TITLE
Org styles

### PR DESCRIPTION
**Pull request by hmarcks**
Completes #100

### CODE REVIEW

**Comment by hmarcks** [comment]
Travis build throws an error, not sure what to do about it...

File: app/views/organizations/_form.html.erb [hunk 1]
```diff
@@ -1,37 +1,38 @@
...
+    </div>
   </div>
 
-  <div class="field">
-    <%= form.label :phone %>
-    <%= form.text_field :phone %>
-  </div>
+  <%= form.button class: "bg-blue white bg-animate hover-bg-dark-blue fw6 br2 pv3 ph4 mr4 mb3" %>
+  <%= link_to 'Cancel Update', @organization, class: "link underline hover-blue dib mr4" %>
+  <%= link_to 'Delete', organization, class: "link underline hover-red dib", method: :delete, data: { confirm: 'Are you sure?' } %>
```
Comment (NickSchimek) [inline app/views/organizations/_form.html.erb]: This form _form partial will render when creating new organizations too. Perhaps we can add a conditional here to only show the Delete link if the organization is already created. What do you think?

```ruby
<% if organization.persisted? %>
  <%= link_to 'Delete', organization, class: "link underline hover-red dib", method: :delete, data: { confirm: 'Are you sure?' } %>
<% end %>
```

File: app/views/organizations/_form.html.erb [hunk 2]
```diff
@@ -1,37 +1,38 @@
...
+        <%= form.text_field :phone, class: "pa3 br2 b--black-20 w-100" %>
+      </div>
+    </div>
   </div>
 
-  <div class="field">
-    <%= form.label :phone %>
-    <%= form.text_field :phone %>
-  </div>
+  <%= form.button class: "bg-blue white bg-animate hover-bg-dark-blue fw6 br2 pv3 ph4 mr4 mb3" %>
```
Comment (NickSchimek) [inline app/views/organizations/_form.html.erb]: I'm not familiar with form.button. Does it do the same thing as form.submit?

File: app/views/organizations/index.html.erb [hunk 3]
```diff
@@ -1,29 +1,31 @@
...
-        <td><%= link_to 'Show', organization %></td>
-        <td><%= link_to 'Edit', edit_organization_path(organization) %></td>
-        <td><%= link_to 'Destroy', organization, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
+      <div class="bg-white shadow pa3 pa4-l">
+        <h2 class="f4 fw7 mb3"><%= organization.name %></h2>
+        <div class="cf">
+          <div class="fl-l w-50-l pr4-l">
+            <h3 class="fw7 mb1">Address</h3>
+            <div class="mb3">123 Placeholder Ave<br>Portland, OR 97202</div>
```
Comment (NickSchimek) [inline app/views/organizations/index.html.erb]: This is excellent 💫 I created issue #103 to get this information added to the db.

File: app/views/organizations/index.html.erb [hunk 4]
```diff
@@ -1,29 +1,31 @@
...
+            <h3 class="fw7 mb1">Dropoff / pickup hours</h3>
+            <div class="mb3">Monday–Friday 9–5</div>
+            <h3 class="fw7 mb1">Contact</h3>
+            <div class="mb1"><%= organization.contact %></div>
+            <div class="mb1"><%= organization.email %></div>
+            <div class="mb3"><%= organization.phone %></div>
+          </div>
+          <div class="fl-l w-50-l">
+            <h3 class="fw7 mb1">Additional Information</h3>
+            <div class="mb4">Additional info placeholder.</div>
```
Comment (NickSchimek) [inline app/views/organizations/index.html.erb]: Created issue #104 to get this org hours and info added to db 👍

**Comment by NickSchimek** [comment]
> Travis build throws an error, not sure what to do about it... Edit - fixed itself?

I tend to not like view tests because they are flakey. When I do write view tests I try to make them flexible because they do have some value.

**Comment by NickSchimek** [comment]
Thanks @hmarcks ! These updates look great. We should move the delete button from the organization form or put it inside the conditional.

File: app/views/organizations/_form.html.erb (hunk 1 above)
Comment (hmarcks) [reply app/views/organizations/_form.html.erb]: My thoughts exactly. I figured that could be addressed when the Admin view is addressed but if it's available I'll add it, thanks!

File: app/views/organizations/_form.html.erb (hunk 2 above)
Comment (hmarcks) [reply app/views/organizations/_form.html.erb]: I just wanted the element to be a <button>. I looked this up and it seems to work...

File: app/views/organizations/index.html.erb (hunk 3 above)
Comment (hmarcks) [reply app/views/organizations/index.html.erb]: Great, thanks!

File: app/views/organizations/index.html.erb (hunk 4 above)
Comment (hmarcks) [reply app/views/organizations/index.html.erb]: Rockin', thank you

**Comment by hmarcks** [comment]
Thanks @NickSchimek, the conditional is now included for Delete

**Pull request merged**